kornia/morphology/basic_operators.py

```diff
@@ -0,0 +1,127 @@
+# Libraries
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+# se_to_mask
+def se_to_mask(se: torch.Tensor) -> torch.Tensor:
```

Comment:
I suggest naming this functiona as `_se_to_mask`since it's not part of the main api